### PR TITLE
Basic Scaffolding of media module

### PR DESCRIPTION
**Pull request by vijaynandwani**
Will add more dependencies later. 
Will fix travis.yml once tests are done
I have added newlines at the end of the files as suggested. Please let me know if this is what you wanted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+
+## Introduction
+
+The Media module provides an extensible framework for managing files and multimedia assets, regardless of whether they are hosted on your own site or a 3rd party site.
+
+Media's aim is to solve Drupal's long standing media handling problem.
+
+## Requirements
+
+* Latest dev release of Drupal 8.x.
```
Comment (paranojik) [inline README.md]: Drupal 8.x-dev should not be a dependency, but media_entity is for sure :)

File: media.info.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: Media
+description: 'Media module for Drupal 8'
+type: module
+package: Media
+core: 8.x
+dependencies:
+  - media_entity_image
```
Comment (paranojik) [inline media.info.yml:8]: media_entity is a dependency

Comment (vijaynandwani) [inline media.info.yml:8]: The media_entity_image module lists media_entity as a dependency but not the modules which are required by media_entity such as the entity module. 
So I assumed I had to only list direct dependencies and other dependencies would be handled automatically. 
I'll fix this and send a new pull request.

File: README.md (hunk 1 above)
Comment (vijaynandwani) [inline README.md]: Sure.

**Pull request closed without merging**